modules/eddb/commodities.js

```diff
@@ -142,26 +154,41 @@ function Commodities() {
             })
             .on('data', async json => {
                 stream.pause();
-                try {
-                    await commoditiesModel.findOneAndUpdate(
-                        {
-                            id: json.id
-                        },
-                        json,
-                        {
-                            upsert: true,
-                            runValidators: true
-                        });
-                    recordsUpdated++;
-                } catch (err) {
+                json.updated_at = json.updated_at * 1000;
+                operations.push({
+                                  updateOne: {
+                                    filter: {
+                                      id: json.id,
+//                                      updated_at: { $ne: json.updated_at }
```

Comment:
this is commented out because otherwise, Mongo returns a duplicate key error for every record NOT updated.  It still updates any changed field, eg `0 inserted, 1000 matched, 1 modified, 0 upserted`